02_calculator/calculator.rb

```diff
@@ -1 +1,16 @@
 #write your code here
+def add(num1, num2)
+    add = num1 + num2
```

Comment:
the `add =` is unnecessary here, and potentially confusing since it has the same name as the function.